polytracker/src/polytracker/output.cpp

```diff
@@ -8,12 +8,25 @@
 #include <functional>
 #include <iomanip>
 #include <iostream>
+#include <llvm/IR/Verifier.h>
 #include <mutex>
 #include <optional>
 #include <set>
 #include <sstream>
 #include <string>
 #include <thread>
+//#include <llvm/IRReader/IRReader.h>
+#include <llvm-c/BitReader.h>
+#include <llvm-c/BitWriter.h>
+#include <llvm-c/Core.h>
+#include <llvm-c/Support.h>
+#include <llvm/IR/Constants.h>
+#include <llvm/IR/LLVMContext.h>
+#include <llvm/IR/Module.h>
+#include <llvm/Support/SourceMgr.h>
+
+#define POLY_TEMP_FILE "/tmp/polytracker_temp.binary"
```

Comment:
This is probably fine for now, but we might get some weird bugs if/when a system ever doesn't have a writable `/tmp` directory. It'd probably be better to use `mktemp`.